crates/sui-core/src/execution_cache/writeback_cache.rs

```diff
@@ -930,16 +942,17 @@ impl WritebackCache {
         // unnecessary cache misses
         self.cached
             .transactions
-            .insert(tx_digest, transaction.clone());
-        self.cached
-            .transaction_effects
-            .insert(effects_digest, effects.clone().into());
+            .insert(&tx_digest, PointCacheItem::Some(transaction.clone()));
```

Comment:
random idea - wonder if this could be simplified a bit with an `into()`? (if you impl it)

Reply:
I'd rather be explicit here since into() can get confusing when you have arcs and options mixed